app/assets/javascripts/js.cookies.js

```diff
@@ -0,0 +1,156 @@
+/*!
```

Comment:
Can you put this file in `vendor/assets`?

http://guides.rubyonrails.org/asset_pipeline.html#asset-organization